### PR TITLE
indexer: remove info log

### PR DESCRIPTION
**Pull request by marbar3778**
## Description

Replace info log with Debug.

### CODE REVIEW

File: state/txindex/indexer_service.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ func (is *IndexerService) OnStart() error {
 			if err = is.idr.AddBatch(batch); err != nil {
 				is.Logger.Error("Failed to index block", "height", height, "err", err)
 			} else {
-				is.Logger.Info("Indexed block", "height", height)
+				is.Logger.Debug("Indexed block", "height", height)
```
Comment (alexanderbez) [inline state/txindex/indexer_service.go:70]:
```suggestion
				is.Logger.Debug("indexed block", "height", height)
```

**Review by alexanderbez — APPROVED** [review]
❤️

**Comment by alexanderbez** [comment]
Damnit, automerge.

**Comment by marbar3778** [comment]
> nit: you could argue the validator updates log should remain at INFO. WDYT? It provides useful info, no?

I was thinking the same thing. The reason why I changed it to debug is, if a node operator is sending logs to elastic search or some other log capture system they will most likely get rid of that log. Also this information is readily available in the app layer.

**Comment by alexanderbez** [comment]
Sure 👍

**Pull request merged**